frontend/src/views/SecretMainPage/components/CreateSecretForm/CreateSecretForm.tsx

```diff
@@ -148,16 +176,17 @@ export const CreateSecretForm = ({
               )
             }
           >
-            <FilterableSelect
+            <CreatableSelect
```

Comment:
This only works from inside an environment. We should also add this to the overview create secret modal. 

![CleanShot 2024-11-26 at 18 56 18](https://github.com/user-attachments/assets/cf15e71c-4262-410e-8cae-b0fb8e99e17f)

Reply:
thought I implemented this
good catch!